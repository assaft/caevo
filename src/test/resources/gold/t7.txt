Two months before the attack, a report was sent.