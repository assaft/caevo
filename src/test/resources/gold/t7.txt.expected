ei1,attack
ei2,sent
t1,P2M
l1,ei1,t1,AFTER
l2,ei2,t1,BEFORE
l3,ei1,ei2,AFTER